server/users.ts

```diff
@@ -1480,9 +1480,13 @@ export class User extends Chat.MessageContext {
 			this.registered ? `[registered]` :
 			``;
 	}
-	battlesForcedPublic() {
-		if (!Config.forcedpublicprefixes) return null;
-		for (const prefix of Config.forcedpublicprefixes) {
+	forcedPublic(key: 'modchat' | 'privacy') {
```

Comment:
This should probably be `battleForcedSetting`?

Reply:
Also, putting it in `room-battle` might make maintenance easier than having the functionality spread out over lots of different files.

Reply:
I'll make it `static` - That functionality _is_ needed in other places.